spec/components/sidebar_controls_component_spec.rb

```diff
@@ -94,10 +101,10 @@
         expect(rendered.css("a[href='/items/druid:zt570qh4444/workflows/new']").inner_text).to eq 'Add workflow'
         expect(rendered.css("a[href='/dor/republish/druid:zt570qh4444']").inner_text).to eq 'Republish'
         expect(rendered.css("a.disabled[data-confirm][data-method='delete'][href='/items/druid:zt570qh4444/purge']").inner_text).to eq 'Purge'
-        expect(rendered.css("a[href='/items/druid:zt570qh4444/source_id_ui']").inner_text).to eq 'Change source id'
+        expect(rendered.css("a[href='/items/druid:zt570qh4444/source_id_ui']").size).to eq 0
```

Comment:
I think you can just remove this line, because we test that there are only 11 buttons and we test for all of them.